src/main/resources/locale/locale_de.properties

```
@@ -216,7 +216,7 @@ Commands.Party.ExpShare = &7Erfahrung &3({0})
 Commands.Party.Features.Header = -----[]&aFunktionen&c[]-----
 Commands.Party.Header = &c-----[]&aParty&c[]-----
 Commands.Party.Invite = &a- Sende eine Party-Einladung
-Commands.Party.Invite.0 = ALERT: &aDu hast eine Party-Einladung f\u00FCr {0} von {1} erhalten.
+Commands.Party.Invite.0 = Warnung: &aDu hast eine Party-Einladung f\u00FCr {0} von {1} erhalten.
```

I think removing the caps here takes away from the artistic choice.
```suggestion
Commands.Party.Invite.0 = ACHTUNG: &aDu hast eine Party-Einladung f\u00FCr {0} von {1} erhalten.
```

---

Not a big fan of using caps, since I don't like being screamed at, not even when it's a warning - but in this case it might be acceptable.

---

I agree with you.
But in my opinion the goal of translating should be to preserve the meaning and style which this includes.